### PR TITLE
[Feature] Add NAT Gateway static IP Flag

### PR DESCRIPTION
**Pull request by AlanLonguet**
**Description** :

This PR introduces a new flag : `static_ip_nat`
This flag controls if we create a static address for the NAT Gateway

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -68,13 +68,27 @@ resource "google_compute_router" "this" {
   ]
 }
 
+resource "google_compute_address" "address_nat" {
```
Comment (JulienJourdain) [inline main.tf:71]: Following Terraform's best practices, you shouldn't repeat "address" word in this resource's name

File: variables.tf [hunk 2]
```diff
@@ -63,6 +63,12 @@ variable "log_config_enable" {
   default     = false
 }
 
+variable "static_ip_nat" {
```
Comment (JulienJourdain) [inline variables.tf:66]: What do you think of:

```suggestion
variable "nat_static_ip" {
```

**Review by JulienJourdain — COMMENTED** [review]
Just few comments.

Do you think that it is interesting to create an output that display NAT IPS? It could be nice to have it in this PR 😀

**Comment by Alan-pad** [comment]
I added the ouput but it's required to get a datasource

**Pull request closed without merging**